src/dx/profiling.c

```diff
@@ -291,6 +292,8 @@ void profiler_print_times() {
 #else
     if (show_profiler) {
 #endif
+        draw_box(0, (WindowStyle)21, 0, 0, 0, SCREEN_WIDTH, SCREEN_HEIGHT, 0x5F, 0, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, NULL,
```

Comment:
In my experience draw_box tends to be slow. Might be better to draw a rect manually here. NBD if that's difficult though

Reply:
You know that's probably true. I don't know how to draw a rect manually, but I can just remove this one for now. I also think it'd be good to use a different way to draw text (maybe how the crash screen does? not sure if that's different) since I noticed even without draw_box that the profiler drops the framerate sometimes.